file/schema.go

```diff
@@ -1,1507 +1,6 @@
-        },
-        "name": {
-          "type": "string"
-        },
-        "path": {
-          "type": "string"
-        },
-        "port": {
-          "type": "integer"
-        },
-        "protocol": {
-          "type": "string"
-        },
-        "read_timeout": {
-          "type": "integer"
-        },
-        "retries": {
-          "type": "integer"
-        },
-        "tags": {
-          "items": {
-            "type": "string"
-          },
-          "type": "array"
-        },
-        "tls_verify": {
-          "type": "boolean"
-        },
-        "tls_verify_depth": {
-          "type": "integer"
-        },
-        "updated_at": {
-          "type": "integer"
-        },
-        "url": {
-          "type": "string"
-        },
-        "write_timeout": {
-          "type": "integer"
-        }
-      },
-      "additionalProperties": false,
-      "type": "object",
-      "anyOf": [
-        {
-          "required": [
-            "name"
-          ]
-        },
-        {
-          "required": [
-            "id"
-          ]
-        }
-      ]
-    },
-    "Unhealthy": {
-      "properties": {
-        "http_failures": {
-          "type": "integer"
-        },
-        "http_statuses": {
-          "items": {
-            "type": "integer"
-          },
-          "type": "array"
-        },
-        "interval": {
-          "type": "integer"
-        },
-        "tcp_failures": {
-          "type": "integer"
-        },
-        "timeouts": {
-          "type": "integer"
-        }
-      },
-      "additionalProperties": false,
-      "type": "object"
-    },
-    "Upstream": {
-      "required": [
-        "name"
-      ],
-      "properties": {
-        "algorithm": {
-          "type": "string"
-        },
-        "client_certificate": {
-          "$ref": "#/definitions/Certificate"
-        },
-        "created_at": {
-          "type": "integer"
-        },
-        "hash_fallback": {
-          "type": "string"
-        },
-        "hash_fallback_header": {
-          "type": "string"
-        },
-        "hash_on": {
-          "type": "string"
-        },
-        "hash_on_cookie": {
-          "type": "string"
-        },
-        "hash_on_cookie_path": {
-          "type": "string"
-        },
-        "hash_on_header": {
-          "type": "string"
-        },
-        "healthchecks": {
-          "$ref": "#/definitions/Healthcheck"
-        },
-        "host_header": {
-          "type": "string"
-        },
-        "id": {
-          "type": "string"
-        },
-        "name": {
-          "type": "string"
-        },
-        "slots": {
-          "type": "integer"
-        },
-        "tags": {
-          "items": {
-            "type": "string"
-          },
-          "type": "array"
-        }
-      },
-      "additionalProperties": false,
-      "type": "object"
-    }
-  }
-}`
+import _ "embed" // for embedding only
+
+//go:embed kong_json_schema.json
+var kongJSONSchema []byte
```

Comment:
You can also embed directly as a string, see https://golang.org/pkg/embed. Out of curiosity, why the array of bytes?

Reply:
Done!